### PR TITLE
Copy update on /kubernetes

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Copy update on `/kubernetes`

## QA

- Go to `/kubernetes` or https://ubuntu-com-11554.demos.haus/kubernetes
- Please compare page against [copy doc](https://docs.google.com/document/d/1b018GoKMF3abEFJlUkfAEK2JEhXWICKF4V0dGdTvgJs/edit#heading=h.vh9li3xlnpwq)
- Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes [#5110](https://github.com/canonical-web-and-design/web-squad/issues/5110)

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by davegoddard42** [comment]
Needs logo

**Comment by davegoddard42** [comment]
Azure Arc logo here: https://assets.ubuntu.com/v1/cecd9d0d-AzureArc.svg

File: templates/kubernetes/index.html [hunk 1]
```diff
@@ -441,7 +476,8 @@ <h2>Kubernetes resources</h2>
           <p class="p-heading--4">Whitepaper</p>
         </div>
       </div>
-      <p><a class="p-link--inverted" href="https://ubuntu.com/engage/bare-metal-kubernetes">How to run workloads on bare metal Kubernetes with MAAS</a></p>
+      <p><a class="p-link--inverted" href="/engage/bare-metal-kubernetes">How to run workloads on bare metal Kubernetes with MAAS</a></p>
+      <p><a class="p-link--inverted" href="/kubernetes/resources">Find more resoureces&nbsp;&rsaquo;</a></p>
```
Comment (mtruj013) [inline templates/kubernetes/index.html:480]:
```suggestion
      <p><a class="p-link--inverted" href="/kubernetes/resources">Find more resources&nbsp;&rsaquo;</a></p>
```

**Review by mtruj013 — APPROVED** [review]
LGTM! Just a typo :)

**Pull request merged**